packages/gov-app/libs/components/WalletConnect.tsx

```diff
@@ -0,0 +1,80 @@
+import type { ChangeEventHandler, FC, MouseEventHandler } from "react";
```

Comment:
good idea, but we should treat this as an enhanced form field, say `WalletField` which should contain only the `Select` and all of the logic to have it working as a standard form field.

The rest of the stuff, e.g the `h2` and `p` contents should belong to the page, as we cannot guarantee all pages will have to use the exact same content.